docs/configuredgraphfactory.txt

```diff
@@ -0,0 +1,466 @@
+[[configuredgraphfactory]]
+ConfiguredGraphFactory
+----------------------
+
+The `ConfiguredGraphFactory` is an access point to your graphs, similar
+to the `JanusGraphFactory`.
```

Comment:
[`GraphFactory`](https://github.com/apache/tinkerpop/blob/3.2.5/gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/structure/util/GraphFactory.java#L38) is a final class in TinkerPop, no interface either, so we're a bit stuck there.